common/service/dynamicconfig/constants.go

```diff
@@ -232,6 +233,8 @@ var keys = map[Key]string{
 	ReplicationTaskProcessorNoTaskInitialWait:              "history.ReplicationTaskProcessorNoTaskInitialWait",
 	ReplicationTaskProcessorCleanupInterval:                "history.ReplicationTaskProcessorCleanupInterval",
 	ReplicationTaskProcessorCleanupJitterCoefficient:       "history.ReplicationTaskProcessorCleanupJitterCoefficient",
+	RPCReplicationEnabled:                                  "history.RPCReplicationEnabled",
+	KafkaReplicationEnabled:                                "history.KafkaReplicationEnabled",
```

Comment:
We named it `XXXEnabled` instead of `EnableXXX` to avoid conflict with the one for frontend? If that's the case probably append the flag name with history/frontend/worker.

Reply:
Updated